src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: Avenir;
+  src: url("/fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("/fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 19px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.bigimg {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  bottom: 26px;
+  z-index: 1;
+  height: 19px;
+  width: 19px;
+}
+
+.smallimg {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  bottom: 16px;
+  z-index: 1;
+  height: 11px;
+  width: 11px;
+}
+
+.searchbar {
+  position: absolute;
+  outline: none;
+  border: none;
+  font-family: Avenir, sans-serif;
+}
+
+.big {
+  position: relative;
+  left: 62px;
+  height: 70px;
+  font-size: 16px;
+}
+
+.small {
+  position: relative;
+  left: 33px;
+  height: 42px;
+  font-size: 14px;
+}
+
+.outer:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.outer:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
```

Comment:
if you set box-sizing in *, no need to set it somewhere else, it's already everywhere